### PR TITLE
Add tvOS libuv branch using unreleased libuv commit.

### PR DESCRIPTION
**Pull request by corvino**
This is for my tvOS fork. It needs the libuv-1.x-tvOS branch from my fork as well, but I'm not sure how include that in the pull request.

### CODE REVIEW

File: mason.sh [hunk 1]
```diff
@@ -434,6 +453,34 @@ function mason_build {
...
+    elif [ ${MASON_PLATFORM} = 'tvos' ]; then
+        mason_substep "Building for Simulator..."
+        export CFLAGS="${MASON_TVSIM_CFLAGS}"
+        cd "${MASON_BUILD_PATH}"
+        mason_compile
+        cd "${MASON_PREFIX}"
+        mv lib lib-tvsim
+        for i in lib-tvsim/*.a ; do lipo -info $i ; done
+
+        mason_substep "Building for iOS..."
```
Comment (1ec5) [inline mason.sh]: This should say “tvOS”.

**Comment by 1ec5** [comment]
https://github.com/corvino/mason/tree/libuv-1.x-tvOS

**Comment by corvino** [comment]
I've got a Mapbox pull request to go along with this; laptop died at a conference before I finished refining it, though. And hotel wifi isn't supporting testing. I'll open the pull request and add a comment later in the weekend.

**Comment by corvino** [comment]
This is in support of https://github.com/mapbox/mapbox-gl-native/pull/2778.

**Comment by jfirebaugh** [comment]
Closing; downstream PR was closed, and if we add support for tvOS in mapbox-gl-native, it will no longer require libuv. That dependency was removed from Darwin-based platforms.

**Pull request closed without merging**